src/Common/Common.Api/Gateways/Entities/UpdateGatewayInstallersRequest.cs

```diff
@@ -0,0 +1,27 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.Serialization;
+
+namespace Microsoft.PowerBI.Common.Api.Gateways.Entities
+{
+    [DataContract]
+    public sealed class UpdateGatewayInstallersRequest
+    {
+        [Required]
```

Comment:
I think you should use IsRequired over the Required attribute.

https://docs.microsoft.com/en-us/dotnet/api/system.runtime.serialization.datamemberattribute.isrequired?view=netframework-4.8
"Gets or sets a value that instructs the serialization engine that the member must be present when reading or deserializing."
vs.
https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.requiredattribute?view=netframework-4.8

Reply:
This is another instance of copying the backend code for better or worse :-(